src/pages/createCohort/components/cohortList.jsx

```diff
@@ -5,8 +5,9 @@ import { Link } from 'react-router-dom';
 import client from '../../../utils/client';
 import './style.css';
 
-const CohortList = () => {
+const CohortList = ({ header }) => {
   const [cohorts, setCohorts] = useState([]);
+  console.log(cohorts);
```

Comment:
Let's remove the debug log before merging

Reply:
Apologies, removed